src/Network/Network/help/Test-AzNetworkWatcherIPFlow.md

```diff
@@ -46,7 +46,7 @@ The Test-AzNetworkWatcherIPFlow cmdlet, for a specified VM resource and a packet
 $nw = Get-AzResource | Where {$_.ResourceType -eq "Microsoft.Network/networkWatchers" -and $_.Location -eq "WestCentralUS" } 
 $networkWatcher = Get-AzNetworkWatcher -Name $nw.Name -ResourceGroupName $nw.ResourceGroupName 
 $VM = Get-AzVM -ResourceGroupName testResourceGroup -Name VM0 
-$Nics = Get-AzNetworkInterface | Where {$_.Id -eq $vm.NetworkInterfaceIDs.ForEach({$_})}
+$Nics = Get-AzNetworkInterface | Where {$_.Id -eq $vm.NetworkProfile.NetworkInterfaces.id.ForEach({$_})}
```

Comment:
Thank you for your find! But for me `$vm.NetworkProfile.NetworkInterfaces.id.ForEach({$_})` returns null so it doesn't filter NICs properly. Not sure why the example uses `ForEach` in the first place.
Could you please change expression in script block to `$vm.NetworkProfile.NetworkInterfaces.Id.IndexOf($_.Id) -ne -1`?

Reply:
@fearofweapons @number213 it may be more clear if we used the `-contains` operator:

```powershell
$Nics = Get-AzNetworkInterface | Where-Object { $vm.NetworkProfile.NetworkInterfaces.Id -contains $_.Id }
```